crosscluster/logical/txnapply: TestTxnApplierIndependent failed

crosscluster/logical/txnapply.TestTxnApplierIndependent [failed](https://mesolite.cluster.engflow.com/invocations/default/5268020f-97e2-4094-91c9-5a53dc8cd6ae?testReportRun=1&testReportShard=1&testReportAttempt=1#targets-Ly9wa2cvY3Jvc3NjbHVzdGVyL2xvZ2ljYWwvdHhuYXBwbHk6dHhuYXBwbHlfdGVzdA==) on master @ [4559deed7a69713aa0874bd436e5453786517102](https://github.com/cockroachdb/cockroach/commits/4559deed7a69713aa0874bd436e5453786517102):


```
    txn_applier_test.go:51: applied txn at 0.000000001,0
    txn_applier_test.go:51: applied txn at 0.000000002,0
    txn_applier_test.go:182: frontier advanced to 0.000000001,0
    txn_applier_test.go:182: frontier advanced to 0.000000002,0
    txn_applier_test.go:51: applied txn at 0.000000001,0
    txn_applier_test.go:51: applied txn at 0.000000002,0
    txn_applier_test.go:182: frontier advanced to 0.000000001,0
    txn_applier_test.go:182: frontier advanced to 0.000000002,0
    txn_applier_test.go:51: applied txn at 0.000000001,0
    txn_applier_test.go:182: frontier advanced to 0.000000001,0
    txn_applier_test.go:51: applied txn at 0.000000002,0
    txn_applier_test.go:182: frontier advanced to 0.000000002,0
    txn_applier_test.go:51: applied txn at 0.000000001,0
    txn_applier_test.go:182: frontier advanced to 0.000000001,0
    txn_applier_test.go:51: applied txn at 0.000000002,0
    txn_applier_test.go:182: frontier advanced to 0.000000002,0
    txn_applier_test.go:51: applied txn at 0.000000001,0
    txn_applier_test.go:182: frontier advanced to 0.000000001,0
    txn_applier_test.go:51: applied txn at 0.000000002,0
    txn_applier_test.go:182: frontier advanced to 0.000000002,0
    txn_applier_test.go:51: applied txn at 0.000000001,0
    txn_applier_test.go:182: frontier advanced to 0.000000001,0
    txn_applier_test.go:51: applied txn at 0.000000002,0
    txn_applier_test.go:182: frontier advanced to 0.000000002,0
    txn_applier_test.go:51: applied txn at 0.000000001,0
    txn_applier_test.go:182: frontier advanced to 0.000000001,0
    txn_applier_test.go:51: applied txn at 0.000000002,0
    txn_applier_test.go:182: frontier advanced to 0.000000002,0
    txn_applier_test.go:51: applied txn at 0.000000001,0
    txn_applier_test.go:182: frontier advanced to 0.000000001,0
    txn_applier_test.go:51: applied txn at 0.000000002,0
    txn_applier_test.go:182: frontier advanced to 0.000000002,0
    txn_applier_test.go:51: applied txn at 0.000000001,0
    txn_applier_test.go:182: frontier advanced to 0.000000001,0
    txn_applier_test.go:51: applied txn at 0.000000002,0
    txn_applier_test.go:182: frontier advanced to 0.000000002,0
    txn_applier_test.go:51: applied txn at 0.000000001,0
    txn_applier_test.go:182: frontier advanced to 0.000000001,0
    txn_applier_test.go:51: applied txn at 0.000000002,0
    txn_applier_test.go:182: frontier advanced to 0.000000002,0
    txn_applier_test.go:51: applied txn at 0.000000001,0
    txn_applier_test.go:51: applied txn at 0.000000002,0
    txn_applier_test.go:182: frontier advanced to 0.000000001,0
    txn_applier_test.go:182: frontier advanced to 0.000000002,0
    txn_applier_test.go:325: 
        	Error Trace:	pkg/crosscluster/logical/txnapply/txn_applier_test.go:325
        	Error:      	"100" is not less than "100"
        	Test:       	TestTxnApplierIndependent
        	Messages:   	expected both orderings within 100 iterations
--- FAIL: TestTxnApplierIndependent (0.08s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>deadlock=true</code>
 - <code>run=1</code>
 - <code>shard=1</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/cdc
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTxnApplierIndependent.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>


Jira issue: CRDB-61073

--- a/pkg/crosscluster/logical/txnapply/txn_applier_test.go
+++ b/pkg/crosscluster/logical/txnapply/txn_applier_test.go
@@ -36,7 +36,7 @@ type testWriter struct {
 func (w *testWriter) ApplyBatch(
 	ctx context.Context, txns []ldrdecoder.Transaction,
 ) ([]txnwriter.ApplyResult, error) {
-	delay := time.Duration(w.rng.Int63n(int64(10 * time.Microsecond)))
+	delay := time.Duration(w.rng.Int63n(int64(100 * time.Microsecond)))
 	select {
 	case <-time.After(delay):
 	case <-ctx.Done():

